property_table.py

```diff
@@ -1030,6 +1036,16 @@ class PropertyTable:
             "FMantissa9",
             True,
         ),
+        "Vmax_soft": (
+            "MaximumCircularVelocitySoft",
+            1,
+            np.float32,
+            "km/s",
+            "Maximum circular velocity when accounting for smoothing length.",
```

Comment:
Probably should be softening rather than smoothing?

Reply:
Changed!